tests/integration/src/xvm.rs

```diff
@@ -330,10 +330,6 @@ fn calling_evm_payable_from_wasm_works() {
         let wasm_caller_addr = deploy_wasm_contract(CALL_EVM_PAYBLE_NAME);
 
         let value = UNIT;
-
-        // claim the default mappings for wasm contract
-        claim_default_accounts(wasm_caller_addr.clone());
-
```

Comment:
Can we add a check for `AccountClaimed` event for the first payable call to make sure the mappings? thanks

Reply:
It's an integration test, which shouldn't assume how it works under the hood. It's supposed to only check balances to make sure the payable call worked.